packages/klighd-core/src/proxy-view/proxy-view.tsx

```diff
@@ -0,0 +1,678 @@
+            // Update bounds
+            clone.bounds = transform;
+
+            // Check if synthesis has specified a proxy rendering
+            if (this.useSynthesisProxyRendering && node.properties && node.properties[ProxyView.PROXY_RENDERING_PROPERTY]) {
+                // Proxy rendering available
+                // console.log("node"); // FIXME:
+                // console.log(node.properties[ProxyView.PROXY_RENDERING_PROPERTY]); // FIXME:
+                // console.log("proxyRendering"); // FIXME:
+                // console.log(node.properties[ProxyView.PROXY_RENDERING_PROPERTY + "2"]); // FIXME:
+                clone.data = node.properties[ProxyView.PROXY_RENDERING_PROPERTY] as KGraphData[];
+            } else {
+                // Fallback, use universal proxy rendering
+                // TODO: further specify what to change for the mock?
+            }
+
+            vnode = ctx.renderProxy(clone);
+
+            // OLD: code to make a proxy non-click-through
+            if (vnode && vnode.data && vnode.data.attrs) {
+                const clickThrough = true;
+                if (!clickThrough) {
+                    vnode.data.attrs["style"] = "pointer-events: auto; " + (vnode.data.attrs["style"] ?? "");
+                }
+            }
+        }
+
+        if (vnode) {
+            // Place proxy at the calculated position
+            this.updateTransform(vnode, transformString);
+            // Store this node
+            this.renderings.set(id, vnode);
+        }
+
+        return vnode;
+    }
+
+    //////// General helper methods ////////
+
+    /** Appends "-proxy" to the given id if the given id isn't already a proxy's id. */
+    private getProxyId(id: string): string {
+        return id.endsWith("-proxy") ? id : id + "-proxy";
+    }
+
+    /** Returns whether the given `node` is valid for rendering. */
+    private canRenderNode(node: SKNode): boolean {
+        if (this.useSynthesisProxyRendering) {
+            return !this.useSynthesisProxyRendering || (node.properties[ProxyView.RENDER_NODE_AS_PROXY_PROPERTY] as boolean ?? true);
+        }
+        // FIXME: remove if and else once the synthesis proxy properties fully work
+        else {
+            // Not an edge, not a comment/non-explicitly specified region
+            // Don't just use includes("$$") since non-explicitly specified regions may contain nodes
+            return node instanceof SKNode && node.id.charAt(node.id.lastIndexOf("$") - 1) !== "$";
+        }
+    }
+
+    /** Updates the proxy-view options specified in the {@link RenderOptionsRegistry}. */
+    updateOptions(renderOptionsRegistry: RenderOptionsRegistry): void {
+        this.prevProxyViewEnabled = this.proxyViewEnabled;
+        this.proxyViewEnabled = renderOptionsRegistry.getValue(ProxyViewEnabled);
+
+        const toPercent = 0.01;
```

Comment:
fromPercent, as you convert the value from a percentage to a real numbered value?

Reply:
Or you convert an integer to an actual percentage

Reply:
The integer (e.g. 5 (%)) is the percentage, the real-numbered value 0.05 is not a percentage